scripts/ble_library.py

```diff
@@ -193,16 +193,20 @@ def copy_hci_files(src_repo_path, dest_lib_path, stm32_serie):
                     sys.exit()
 
 
-def update_ble_lib_readme(lib_path, make_version, make_commit):
+def update_ble_lib_readme(lib_path, make_version, make_commit, stm32_serie):
     """Update README file
 
     Args:
         dest_lib_path: library path
         make_version: latest STM32Cube version.
         make_commit: Commit corresponding to latest STM32Cube version.
+        stm32_serie: stm32 serie
     """
 
-    readme_path = Path(lib_path / "hci" / "README")
+    if stm32_serie == "stm32wba":
+        readme_path = Path(lib_path / "README")
+    else:
+        readme_path = Path(lib_path / "hci" / "README")
```

Comment:
Note: For this last change, I'd actually prefer we move the WB README file to align it on WBA/WB0